docs/apidoc.md

```diff
@@ -0,0 +1,16 @@
+# juCi++ API doc
+
+## Prerequisites:
+ * doxygen
+ * plantuml (install via apt-get or download from http://de.plantuml.com/; if downloaded either copy the jar file to /usr/bin or set the environment variable PLANTUML_PATH to point to the path containing the jar file)
+
```

Comment:
Notes:
* Link to install instructions provided by platuml instead. java also dependency?
* plantuml generates uml, not needed for api-only build.